instantswap/exchange/trocador/trocador.go

```diff
@@ -184,10 +184,10 @@ func (t *trocador) CreateOrder(vars instantswap.CreateOrder) (res instantswap.Cr
 		ChargedFee:     0,
 		Destination:    trade.AddressUser,
 		ExchangeRate:   trade.rate(),
-		FromCurrency:   trade.CoinFrom,
+		FromCurrency:   strings.ToUpper(trade.CoinFrom),
```

Comment:
i decided to log the from and to currencies

`FromCurrency: DECRED, ToCurrency: LITECOIN (MAINNET - NO MW)`

while the app checks for DCR and LTC, hence why it is stil showing unsupported asset, we would need to handle this edge case

Reply:
we could pass the coin names through a method that renames them appropriately

Reply:
https://github.com/crypto-power/instantswap/pull/10/commits/5fc1bfb43ecdf495c9b2a43e26f948312aae99d2
Fixed here

Reply:
i'd create a follow up issue for the trocador error message not being clear enough